SPECIFICATION.md

```diff
@@ -253,13 +254,31 @@ three different options which allow to control letter spacing:
    Glyphs can also adjust the trailing space by using the `~` sign for
    specifying additional spacing (see section "Whitespaces" above).
 
-The total space between two letters is calculated by the addition of these three
-parameters:
+### 4.1.1. Proportional Fonts
+
+The total space between two letters of a proportional (non-monospace) font is
+calculated by the addition of these three parameters:
 
 1. Trailing space of first letter (`~` space definition)
 2. Global letter spacing (`letter_spacing` from header)
 3. Leading space (leftmost X-coordinate) of second letter
 
+### 4.1.2. Monospace Fonts
+
+If a font specifies the `monospace_width` header (see subsection "Standardized
+Headers"), then that font SHOULD be layouted as a monospace font. The value of
+the `monospace_width` header corresponds to the width of every glyph's bounding
+box, excluding global spacing.
```

Comment:
Maybe "excluding global spacing" is not clear enough that the global `letter_spacing` is applied after every glyph?